docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,392 @@
+  # in cases where the client's utility functions will be used (e.g., the
+  # relay subcommand).
+  KeepRandomBeaconService = "0xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"
+
+# Keep network configuration.
+[LibP2P]
+  Peers = ["/ip4/127.0.0.1/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1", "/dns4/some-keep-host.com/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1"]
+  Port  = 3920
+  # Override the node's default addresses announced in the network
+  AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
+
+[Storage]
+  DataDir = "/my/secure/location"
+```
+
+==== Parameters
+
+[%header,cols=4*]
+|===
+|`ethereum`
+|Description
+|Default
+|Required
+
+|`URL`
+|The Ethereum host your keep-client will connect to.  Websocket protocol/port.
+|""
+|Yes
+
+|`URLRPC`
+|The Ethereum host your keep-client will connect to.  RPC protocol/port.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.account`
+|Description
+|Default
+|Required
+
+|`Address`
+|The Keep operator Ethereum account address.
+|""
+|Yes
+
+|`KeyFile`
+|The local filesystem path to your Keep operator Ethereum account keyfile.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.ContractAddresses`
+|Description
+|Default
+|Required
+
+|`KeepRandomBeaconOperator`
+|Hex-encoded address of the KeepRandomBeaconOperator Contract.
+|""
+|Yes
+
+|`KeepRandomBeaconService`
+|Hex-encoded address of the KeepRandomBeaconService Contract.
+|""
+|Yes
+
+|`TokenStaking`
+|Hex-encoded address of the TokenStaking Contract.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`LibP2P`
+|Description
+|Default
+|Required
+
+|`Peers`
+|Comma separated list of network peers to boostrap against.
+|[""]
+|No
+
+|`Port`
+|The port to run your instance of Keep on.
+|3919
+|Yes
+
+|`AnnouncedAddresses`
+|Multiaddr formatted hostnames or addresses annouced to the Keep Network. More on multiaddr format https://docs.libp2p.io/reference/glossary/#multiaddr[here].
+|[""]
+|No
+|===
+
+[%header,cols=4*]
+|===
+|`Storage`
+|Description
+|Default
+|Required
+
+|`DataDir`
+|Location to store the Keep nodes group membership details.
+|""
+|Yes
+|===
+
+== Deployment
+
+Deployment constraints are as follows:
+
+- Each beacon client deployment requires a unique operator address.
+- Each beacon client deployment requires a unique IP address.
+
+=== Kubernetes
+
+At Keep we run on GCP + Kube. To accommodate the aforementioned constraints and system
+recommendations we use the following pattern for each of our environments:
+
+- Regional Kube cluster.
+- 5 beacon clients, each running minimum stake required by the network.
+- A LoadBalancer Service for each client.
+- A StatefulSet for each client.
+
+You can see our Ropsten Kube configurations https://github.com/keep-network/keep-core/tree/master/infrastructure/kube/keep-test[here]
+
+== Logging
+
+Below are some of the key things to look out for to make sure you're booted and connected to the
```

Comment:
Referring to your other comment about logging, our logging is configurable to some extent with the following environment variables:
```
LOG_LEVEL=DEBUG
IPFS_LOGGING_FMT=nocolor
GOLOG_FILE=/var/log/keep/keep.log
GOLOG_TRACING_FILE=/var/log/keep/trace.json
```

Reply:
Thanks!